app/views/layouts/_main_nav.html.erb

```diff
@@ -1,6 +1,10 @@
-<% nav_links = NavigationLink.all.ordered.to_a %>
+<% default_nav_links = NavigationLink.default_section.ordered.to_a %>
+<% other_nav_links = NavigationLink.other_section.ordered.to_a %>
 
-<nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" aria-label="Primary sidebar nav">
+<nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" aria-labelledby="<%= community_name %>">
+  <h2 class="mb-1 fs-base">
+    <%= community_name %>
+  </h2>
```

Comment:
```suggestion
<nav class="mb-6 <% unless user_signed_in? %>mt-4<% end %>" aria-labelledby="community-nav-heading">
  <h2 id="community-nav-heading" class="mb-1 fs-base">
    <%= community_name %>
  </h2>
```

Just a small tweak - `aria-labelledby` looks for an element with the given ID to use as the label (as opposed to "aria-label" which just takes a text string). We can link it with the label like this ☝️ 

For a bit of context, using aria-labelledby like this instead of aria-label means that if users are translating the page into other languages, the label will be translated too ✨  Wheras aria-label would always be the English text we enter inline.